CHANGELOG.md

```diff
@@ -10,17 +10,13 @@ All versions prior to 0.9.0 are untracked.
 
 ### Added
 
-* API: `Signer.sign()` can now take a `Hashed` as an input,
+* API: `Signer.sign_artifact()` can now take a `Hashed` as an input,
   performing a signature on a pre-computed hash value
   ([#860](https://github.com/sigstore/sigstore-python/pull/860))
```

Comment:
This should really talk about `Signer.sign_artifact()` as a brand new method (and could mention how it's a replacement for removed `Signer.sign()`)